explorer/src/lib/dusk/components/card/styles.css

```diff
@@ -0,0 +1,34 @@
+.dusk-card {
+  background-color: var(--surface-color);
+  border-radius: var(--control-border-radius-size);
+  padding: 1.3em 2em;
```

Comment:
I had a chat about this with @kieranhall some time ago: as we are moving towards a proper design language (where we will have variants, pre-defined gaps, spacings, etc.) and with CSS in the component folder, it's probably better to switch to `rem` units everywhere even now.